### DISCUSSION TITLE
Add User to Webhook Information

### DISCUSSION THREAD

**Discussion by erobertsme** [nocodb/nocodb#7420] [Features]
It would be very useful to include the user that made the action which invoked the webhook. This would allow the possibility to filter webhooks based on users.

**Comment by dstala** [comment]
@erobertsme from release v204.0, NocoDB supports `created-by` `last-modified-by` fields. You can add these to your table to get this information. 
https://docs.nocodb.com/category/user-based

let us know if it solves your usecase.

**Comment by erobertsme** [comment]
That is indeed helpful! It pretty much solves the problem I was having.

It's a shame I was just looking for this right before the update was released.

I still feel like this info could be included by default without having to add the fields, at least to the webhooks. It took quite a while to add all of the fields to each table in the base.
